workers/unity/Packages/com.improbable.gdk.core/Worker/WorkerConnector.cs

```diff
@@ -59,7 +71,14 @@ await Worker.CreateWorkerAsync(GetReceptionistConfig(workerType), logger, origin
                 }
 
                 var worker = await ConnectWithRetries(connectionDelegate, MaxConnectionAttempts, logger, workerType);
-                InitializeWorker(worker);
+
+                Worker = worker;
```

Comment:
I think the `worker` variable is not needed here

Reply:
Yep. Will remove.